### DISCUSSION TITLE
Welcome to bridge. Discussions!

### DISCUSSION THREAD

**Discussion by solvedDev** [bridge-core/bridge.#298] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bananaron** [comment]
how do you install bridge

**Reply by solvedDev** [reply]
See #249

**Reply by personiguess-a** [reply]
I need help. 
I can export as brproject, but when exporting as macaddon, it gets stuck on the loading message forever.

**Comment by Tembigdeal** [comment]
hai, i`m a new person to github and i  saw bridge and i went to Youtube and saw how good it was then i wanted to download it but when i clicked the exe installation I don`t know if i spelled that right but it started the setup and while it was installing it stopped with an error saying my internet connection was unavailable but i obviously have an internet connection otherwise how did i get to visit github at the first place so please help, Thanks.

**Reply by solvedDev** [reply]
We have had a lot of people with download problems and we are not able to fix them because the installer is not something we maintain ourselves. We will have this fixed with bridge. v2 which will ship as a web app though

**Reply by Tembigdeal** [reply]
okay, if thats gonna allow me to use it then ill wait if its a concept of time but question would it change anything if my internet was wireless or not and will it change anything if i changed to an entirely new pc?

**Reply by solvedDev** [reply]
The web app will work just as you would expect it from a native app. On a new computer, you would need to copy your project files and install the app again

**Reply by Tembigdeal** [reply]
alright um yeah certainly thats all i got to ask so is there any estimated time(Date) for when the release of the web app gonna be?(P.S English is not my first language so im very sorry if i mess up while talking never had to talk like a professional before and since im not talking to a person thats the same small brain as me im kind of trying to get my grammer and spelling not to embarrass me)

**Comment by personiguess-a** [comment]
I need help.
I can export as brproject, but when exporting as macaddon, it gets stuck on the loading message forever.
